packages/block-library/src/media-text/media-container.native.js

```diff
@@ -44,6 +44,15 @@ import SvgIconRetry from './icon-retry';
  */
 const ALLOWED_MEDIA_TYPES = [ MEDIA_TYPE_IMAGE, MEDIA_TYPE_VIDEO ];
 
+export function imageFillStyles( url, focalPoint ) {
+	return url ?
+		{
+			backgroundImage: `url(${ url })`,
+			backgroundPosition: focalPoint ? `${ focalPoint.x * 100 }% ${ focalPoint.y * 100 }%` : `50% 50%`,
+		} :
+		{};
+}
+
```

Comment:
Since this is a pure function, I believe we can re-export directly, avoiding the need to copy the implementation:

`export { imageFillStyles } from './media-container.js';`

Reply:
Thanks for testing it and for the suggestion, you're right, better than duplicating it. Updated the PR =)